applications/web-browser/page.js

```
@@ -0,0 +1,223 @@
+		/^(https?|file|ftps?|mailto|javascript|data:image\/[^;]{2,9};):/i.test(
+			url
+		)
+	) {
+		return url; //Url is already absolute
+	}
+	base_url = base_url.match(/^(.+)\/?(?:#.+)?$/)[0] + "/";
+	if (url.substring(0, 2) === "//") {
+		return "https:" + url;
+	} else if (url.charAt(0) === "/") {
+		var matches = base_url.match(/^https?\:\/\/([^\/?#]+)(?:[\/?#]|$)/i);
+		var domain = matches && matches[1];
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)